frontend/demo/component/menubar/react/menu-bar-basic.tsx

```diff
@@ -0,0 +1,39 @@
+import { MenuBar, type MenuBarItem } from '@hilla/react-components/MenuBar.js';
+import { reactExample } from 'Frontend/demo/react-example'; // hidden-source-line
+import React, { useState } from 'react';
```

Comment:
Let's align imports order to match other React examples:

```suggestion
import { reactExample } from 'Frontend/demo/react-example'; // hidden-source-line
import React, { useState } from 'react';
import { MenuBar, type MenuBarItem } from '@hilla/react-components/MenuBar.js';
```